packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -365,6 +366,14 @@ abstract class FlutterCommand extends Command<void> {
       help: 'Build a JIT release version of your app${defaultToRelease ? ' (default mode)' : ''}.');
   }
 
+  void addFontSubsetFlag() {
+    argParser.addFlag('font-subset',
```

Comment:
maybe `tree-shake-fonts`? 

I'm not sure if folks would be able to determinate what this flag does from the current name and help text.